README.md

```
@@ -308,9 +310,17 @@ cd zmsapi
 
 ## Cronjobs
 To run cronjobs locally use ddev
+
+**Hourly cronjob with city-specific flags or default (which is also Berlin but compatible with dldb-mapper for now):**
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)